build/Scripts/PrivateXdt/GeneratePrivateXdt.ps1

```diff
@@ -31,6 +31,7 @@ param(
 )
 
 Import-Module "$PSScriptRoot\XdtHelper.psm1" -Force
+Add-Type -AssemblyName 'System.Xml.Linq'
```

Comment:
nit: Could you make these changes in a separate PR so as not to conflate the two sets of changes?

Reply:
Sure, I'll separate them out as separate PRs